src/historywork/GetHistoryArchiveStateWork.h

```diff
@@ -31,11 +31,23 @@ class GetHistoryArchiveStateWork : public Work
 
   public:
     GetHistoryArchiveStateWork(
```

Comment:
I would be happy to see that change everywhere we are passing references to work constructors (like VerifyBucketWork).

Reply:
Agreed, but it's outside of the scope of this PR. I can open an issue to track this though